src/main/java/de/dafuqs/spectrum/mixin/EffectCommandMixin.java

```diff
@@ -25,6 +26,9 @@ private static void clearIncurableEffects(ServerCommandSource source, Collection
             for (StatusEffectInstance effect : living.getStatusEffects()) {
                 if (((Incurable) effect).spectrum$isIncurable())
                     ((Incurable) effect).spectrum$setIncurable(false);
+                // manually remove fatal slumber to bypass turning it into eternal slumber
+                if (effect.getEffectType() == SpectrumStatusEffects.FATAL_SLUMBER)
+                    living.removeStatusEffect(SpectrumStatusEffects.FATAL_SLUMBER);
```

Comment:
... Why? Can you not just unconditionally call that outside of the loop?

Reply:
... not sure how i missed that lmao